lib/matplotlib/figure.py

```diff
@@ -2420,11 +2420,11 @@ def __init__(self,
                 The use of this parameter is discouraged. Please use
                 ``layout='constrained'`` instead.
 
-        layout : {'constrained', 'compressed', 'tight', `.LayoutEngine`, None}
+        layout : {'constrained', 'compressed', 'tight', 'none', `.LayoutEngine`, \
+None}, default: None
```

Comment:
Its not your fault, but the `'constrained'` bullet below has an extra indent

Reply:
Oh _that's_ why it's in bold.  I was scratching my head about that...